### PR TITLE
update teacher images on congrats

### PR DESCRIPTION
**Pull request by Erin007**
Desktop:
desktop teacher

Third party courses on mobile:
codeorgteacher thridparty

Code.org courses on mobile:
codeorg teacher mobile

### CODE REVIEW

**Comment by breville** [comment]
nit: "Code.org courses on mobile" body text comes too close to grey button.

File: apps/src/templates/VerticalImageResourceCard.jsx [hunk 1]
```diff
@@ -114,6 +114,8 @@ class VerticalImageResourceCard extends Component {
       require('@cdo/static/resource_cards/codeorgteacher.png'),
```
Comment (breville) [inline apps/src/templates/VerticalImageResourceCard.jsx:114]: nit: not sure what our linter says, but this is a bit hard to read with no indentation..

**Comment by breville** [comment]
minor: `thirdpartyteacher.png` has a strip of transparent pixels on its left edge.

**Comment by Erin007** [comment]
This has a green Circle run, so I'm going to merge it now to get these changes in for the bug bash; then, I'll circle back for another polishing round.

**Pull request merged**